testing/opae-cxx/test_events_cxx_core.cpp

```diff
@@ -48,13 +48,15 @@ using namespace opae::fpga::types;
 class events_cxx_core : public ::testing::TestWithParam<std::string> {
  protected:
   events_cxx_core()
-        : tmpfpgad_log_("tmpfpgad-XXXXXX.log"),
-          tmpfpgad_pid_("tmpfpgad-XXXXXX.pid"),
+        : tokens_({nullptr, nullptr}),
           handle_(nullptr) {}
 
   virtual void SetUp() override {
-    tmpfpgad_log_ = mkstemp(const_cast<char *>(tmpfpgad_log_.c_str()));
-    tmpfpgad_pid_ = mkstemp(const_cast<char *>(tmpfpgad_pid_.c_str()));
+    strcpy(tmpfpgad_log_, "tmpfpgad-XXXXXX.log");
+    strcpy(tmpfpgad_pid_, "tmpfpgad-XXXXXX.pid");
+
+    tmpfpgad_log_fd_ = mkstemps(tmpfpgad_log_, 4);
+    tmpfpgad_pid_fd_ = mkstemps(tmpfpgad_pid_, 4);
```

Comment:
I think we can just safely use close here and remove the fds..
ie
  close(mkstemps(tmpfpgad_log_, 4));
  close(mkstemps(tmpfpgad_pid_, 4));